src/content/docs/en/basics/astro-pages.mdx

```diff
@@ -96,10 +96,56 @@ Files with the `.html` file extension can be placed in the `src/pages/` director
 
 ## Custom 404 Error Page
 
-For a custom 404 error page, you can create a `404.astro` or `404.md` file in `/src/pages`.
+For a custom 404 error page, you can create a `404.astro` or `404.md` file in `src/pages`.
 
 This will build to a `404.html` page. Most [deploy services](/en/guides/deploy/) will find and use it.
 
+## Custom 500 Error Page
+
+For a custom 500 error page, you can create a `500.astro` in `src/pages`.
+
+:::caution
+Once deployed, Astro will only use `500.astro` when using `output: 'hybrid'` for pages that opt-out from prerendering or `output: 'server'`.
+:::
+
+If an error occurs in this file, it will default to the host 500 error page.
+
+### Retrieve The Error
+
+<p><Since v="4.10.0" /></p>
+
+`500.astro` accepts an `error` prop. It can be anything, so make sure to handle it properly:
+
+```astro title="src/pages/500.astro"
+---
+interface Props {
+    error: unknown
+}
+
+const { error } = Astro.props
+---
+
+<div>{error instanceof Error ? error.message : 'Unknown error'}</div>
+```
+
+### Usage In Development
```

Comment:
```suggestion
### Enable in development mode
```
I think this sounds more descriptive as to what's actually going on? "Usage in development" made think I was going to read, "Oh, and this is how it's used", not that you can enable/access this page even in dev mode. (Which I think is what you mean?)